hypha/apply/funds/forms.py

```diff
@@ -408,3 +408,28 @@ def __init__(self, *args, **kwargs):
         kwargs.pop('user')
         super().__init__(*args, **kwargs)
         self.fields['meta_terms'].queryset = MetaTerm.get_root_descendants().exclude(depth=2)
+
+
+class CreateReminderForm(forms.ModelForm):
+    submission = forms.ModelChoiceField(
+        queryset=ApplicationSubmission.objects.filter(),
+        widget=forms.HiddenInput(),
+    )
+    time = forms.DateTimeField(input_formats=['%Y-%m-%dT%H:%M'])
+
+    def __init__(self, instance=None, user=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.user = user
+
+        if instance:
+            self.fields['submission'].initial = instance.id
+
+    def save(self, *args, **kwargs):
+        return Reminder.objects.create(
+            time=self.cleaned_data['time'],
+            submission=self.cleaned_data['submission'],
+            user=self.user)
+
+    class Meta:
+        model = Reminder
+        fields = ['time', 'action', 'medium']
```

Comment:
Most reminders will only have one "medium", e-mail or Slack. Reminders to staff will always be Slack but reminders to reviewers will always be e-mail (at least for now).

How is this handled in the UI?

Reply:
Currently, we allow a staff user to pick a medium for each reminder while creating new reminder. And it defaults to Email for now because we have only one action (Remind reviewers to review). But it can be a bad UX when more actions are created like Reminder to Staff. So, the user may pick the wrong medium like email for staff action. For the current requirements and possible future actions, it seems to have medium constant in the code only and do not allow staff user to pick medium. What do you suggest?

Reply:
Yes, keep the "medium" data but do not expose it in the UI. Hard code it to each action in code for now.